blog-service/2023-05-11-apps.md

```diff
@@ -0,0 +1,16 @@
+---
+title: May 11, 2023 (Apps)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - apps
+  - app catalog
+  - saas cloud
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Zero Networks Segment Source. This source ingests audit logs and network activities via Zero Networks Segment API to provide comprehensive network security by implementing zero-trust principles.
+
+For more information, check out our technical documentation [here](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source).
```

Comment:
Truncating because the whole paragraph will likely be captured in RSS feeds this way

```suggestion
We're excited to announce the release of our new cloud-to-cloud source for Zero Networks Segment. This source ingests audit logs and network activities via Zero Networks Segment API to provide comprehensive network security by implementing zero-trust principles. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source).
```